Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM php:5.6-fpm
 MAINTAINER Oleg Kopachovets <ok@procoders.tech>
 
 #install laravel requirements and aditional extensions
-RUN requirements="libmcrypt-dev g++ libicu-dev libmcrypt4 libicu52 zlib1g-dev git libcurl4-openssl-dev" \
```

Comment:
there are no php-gd package at apt repositories.
plz, try to build it locally before any push